### ISSUE TITLE
Add more info about app

### ISSUE THREAD

**Issue by Drsheppard01** [Portable-Linux-Apps/Portable-Linux-Apps.github.io#2]
It would be cool to add lines such as `license`, `last update` and `download link`
You can also add links to `donate`, `translate`, `report a bug` which help developers and mantainers improve app or build

**Comment by ivan-hc** [comment]
This is a grea idea, but there is a point that prevents me to do this: I am alone!

**Comment by Drsheppard01** [comment]
I can add these lines to all the pages, it won't bother me. But will they be automatically recognized by the deplayer and displayed on the site?

**Comment by ivan-hc** [comment]
The urls should be made "clickable", but by doing this we will have a confusing output when we run `am -a PROGRAM` or `am -w PROGRAM`. The best solution would be to add an HTML or other models to made all these links clickable from the web, without having to preack the compatibility in the terminal.

However, I'm not good with HTML and web design and this is my first website in Markdown.

**Comment by ivan-hc** [comment]
Today I've uploaded a script I use to generate https://portable-linux-apps.github.io/apps.html from the list of x86_64 apps:

https://github.com/ivan-hc/AM-Application-Manager/blob/main/tools/am2pla-site

Maybe something similar can be done to update all the pages.

However, I think that using an HTML is the best way, we need someone that know how to do it.

**Comment by ivan-hc** [comment]
@Drsheppard01 today I've discovered that I had to put only `markdown: GFM` in the ¯config.yml file of this repository to made all the tables available in HTML like on the Markdown viewer... and only noy I've seen that this made all the links ***CLICKABLE***!!

In each page we have links to the main pages of the websites, the souces and the download links... all the people are needing to know is there. Donations? They can click on the page of the developer and donate there.

I think that we have solved this issue... casually, but it is solved.

PS: just reached 1596 installation scripts for x86_64 only (about 1550 Appimage packages)!
